example/trace/otlphttp/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/GoogleCloudPlatform/opentelemetry-operations-go/example/trace/otlphttp
 
-go 1.22.0
+go 1.22.7
```

Comment:
looks like this updates the minimum go version in a bunch of modules.  I don't think we want to do that yet

Reply:
The collector [requires go1.23.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/go.mod#L11) which is why a lot of those got updated; I won't be able to put them back and remain on this OTel version

Reply:
If possible, we should drop support for go1.22 after the release happens